src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/status/MethodologyStatusPage.tsx

```diff
@@ -34,6 +36,8 @@ const MethodologyStatusPage = () => {
     onMethodologyChange,
   } = useMethodologyContext();
 
```

Comment:
We could use destructing to make this a little bit more succinct (similar to what we do in other places): 

`const { PublicAppUrl } = useConfig`